index.html

```diff
@@ -314,6 +315,39 @@ <h6>CSS</h6>
       </div>
     </section>
 
+    <section class="contact-me">
+      <div class="contact-form">
```

Comment:
- [ ] Kindly make sure the width of the about and the from section are a `100%` and fit all the screen in large
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193897554-9d738d6b-2c59-4441-a274-2aec8f8f4069.png)
![image](https://user-images.githubusercontent.com/97100091/193897840-29513bf4-0881-4991-9b85-d808c3888b5f.png)


The Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193897653-3d33f848-d15f-41c6-bbaf-1b970f464703.png)
![image](https://user-images.githubusercontent.com/97100091/193897770-195c7e50-de01-4adb-bf43-cbe1b392d33a.png)